src/components/CvAccordion/CvAccordion.vue

```diff
@@ -83,6 +83,19 @@ provide('registerItem', (itemCvId, item) => {
 provide('deregisterItem', itemCvId => {
   items.delete(itemCvId);
 });
+
+/**
+ * @typedef {Object} AccordianChangeElement
```

Comment:
side note: typo in Accordion word in whole types section
is: Accordi**a**n     =>     should be: Accordi**o**n

Reply:
Nice find! Thanks!